aea/cli/core.py

```diff
@@ -193,6 +197,100 @@ def add_key(ctx: Context, type_, file):
     )
 
 
+@cli.command()
+@click.argument(
+    "type_",
+    metavar="TYPE",
+    type=click.Choice(
+        [DefaultCrypto.identifier, FetchAICrypto.identifier, EthereumCrypto.identifier]
+    ),
+    required=True,
+)
+@pass_ctx
+def get_address(ctx: Context, type_):
+    """Get the address associated with the private key."""
+    try_to_load_agent_config(ctx)
+
+    _verify_or_create_private_keys(ctx)
+    private_key_paths = dict(
+        [
+            (identifier, config.path)
+            for identifier, config in ctx.agent_config.private_key_paths.read_all()
+        ]
+    )
+    try:
+        wallet = Wallet(private_key_paths)
+        address = wallet.addresses[type_]
+        logger.info("{}".format(address))
+    except ValueError as e:  # pragma: no cover
+        logger.error(str(e))  # pragma: no cover
+
+
+@cli.command()
+@click.argument(
+    "type_",
+    metavar="TYPE",
+    type=click.Choice([FetchAICrypto.identifier, EthereumCrypto.identifier]),
+    required=True,
+)
+@pass_ctx
+def get_wealth(ctx: Context, type_):
+    """Get the wealth associated with the private key."""
+    try_to_load_agent_config(ctx)
+
+    _verify_or_create_private_keys(ctx)
+    private_key_paths = dict(
+        [
+            (identifier, config.path)
+            for identifier, config in ctx.agent_config.private_key_paths.read_all()
+        ]
+    )
+    wallet = Wallet(private_key_paths)
+    try:
+        _verify_ledger_apis_access()
+        ledger_api_configs = dict(
+            [
+                (identifier, cast(Dict[str, Union[str, int]], config))
+                for identifier, config in ctx.agent_config.ledger_apis.read_all()
+            ]
+        )
+        ledger_apis = LedgerApis(ledger_api_configs, ctx.agent_config.default_ledger)
+
+        address = wallet.addresses[type_]
+        balance = ledger_apis.token_balance(type_, address)
+        logger.info("The wealth for address {} is: {}".format(address, balance))
```

Comment:
the same arguments about the prefix hold here.

This is the output I get:
![Screenshot from 2020-01-25 12-14-18](https://user-images.githubusercontent.com/9467617/73120287-3d8ead80-3f6c-11ea-8cf7-e80c462db5ae.png)

This case is different because there's also the address, so the prefix of the output brings some useful information.

However, do I really need that? After all, I just want the amount of wealth in my wallet. The address is identified by the `type_` parameter, so I can do `aea get-address` and get it.